templates/frontend/docker-compose_template.erb

```diff
@@ -88,6 +105,12 @@ services:
             <% if @statsd_enabled == true -%>--statsd_host <%= @statsd_host %> --statsd_prefix sarimner.<%= @instance %><% end -%>
             --stats_url /var/run/haproxy-control/stats
             'site=<%= @site_name %>;group=default'
+<% if @dns.is_a? Array -%>
+    dns:
```

Comment:
jag hade nog testat för is_array && len > 0 för att vara säker på att inte putta ut dns: taggen om man råkar sätta en tom array

Reply:
Should be fixed in https://github.com/SUNET/puppet-sunet/pull/12/commits/e729ac21680d98ba6b2d888012959223e5b14467